Assignment2/manumazvi-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": {
+    "prefix": <http://example.org/elements>,
+    "xsd": <http://www.w3.org/2001/XMLSchema#>

```

Comment:
They shouldn't be defined between <>
Also, there is something missing at the end of the URI of "prefix"